src/classes/Commands.ts

```diff
@@ -1546,26 +1549,41 @@ export = class Commands {
 
             this.bot.sendMessage(steamID, 'Accepting offer...');
 
+            const partnerId = new SteamID(this.bot.manager.pollData.offerData[offerIdString].partner);
+            const reply = offerIdAndMessage.substr(offerIdString.length);
+            const adminDetails = this.bot.friends.getFriend(steamID);
+
             this.bot.trades.applyActionToOffer('accept', 'MANUAL', {}, offer).asCallback(err => {
                 if (err) {
                     this.bot.sendMessage(
                         steamID,
                         'Ohh nooooes! Something went wrong while trying to accept the offer: ' + err.message
                     );
+                    return;
+                }
+                // Send message to recipient if includes some messages
+                if (reply) {
+                    this.bot.sendMessage(
+                        partnerId,
+                        'Message from ' + (adminDetails ? adminDetails.player_name : 'admin') + ': ' + reply
+                    );
```

Comment:
Instead of doing this then maybe make a function for admins to send messages to people.

Reply:
You mean need to adjust the "!message" command?
Btw this "!accepttrade <offerID> [message]" or "!declinetrade <offerID> [message]" is just one time accept or decline note to trade partner.